docs/integrations/graphite.md

```diff
@@ -0,0 +1,114 @@
+
+## Overview
+
+M3 supports ingesting Graphite metrics using the [Carbon plaintext protocol](https://graphite.readthedocs.io/en/latest/feeding-carbon.html#the-plaintext-protocol). We also support a variety of aggregation and storage policies for the ingestion pathway (similar to [storage-schemas.conf](https://graphite.readthedocs.io/en/latest/config-carbon.html#storage-schemas-conf) when using Graphite Carbon) that are documented below. Finally, on the query side, we support the majority of [graphite query functions](https://graphite.readthedocs.io/en/latest/functions.html).
+
+## Ingestion
+
+Setting up the M3 stack to ingest carbon metrics is straightforward. First, make sure you've followed our [other documentation](../how_to/single_node.md) to get m3coordinator and M3DB setup. Also, familiarize yourself with how M3 [handles aggregation](../how_to/query.md).
+
+Once you have both of those services running properly, modify your m3coordinator configuration to add the following lines and restart it:
+
+```yaml
+carbon:
+  ingester:
+    listenAddress: "0.0.0.0:7204"
+```
+
+This will enable a line-based TCP carbon ingestion server on the specified port. By default, the server will write all carbon metrics to every aggregated namespace specified in the m3coordinator [configuration file](../how_to/query.md) and aggregate them using a default strategy of `mean` (equivalent to Graphite's `Average`).
+
+This default setup makes sense if your carbon metrics are unaggregated, however, if you've already aggregated your data using something like [statsite](https://github.com/statsite/statsite) then you may want to disable M3 aggregation. In that case, you can do something like the following:
+
+```yaml
+carbon:
+  ingester:
+    listenAddress: "0.0.0.0:7204"
+    rules:
+      - pattern: .*
+        aggregation:
+          enabled: false
+        policies:
+          - resolution: 1m
+            retention: 48h
+```
+
+This replaces M3's default behavior with a single rule which states that all metrics (since `.*` will match any string) should be written to whichever aggregated M3DB namespace has been configured with a resolution of `1 minute` and a retention of `48 hours`, bypassing aggregation / downsampling altogether. Note that there *must* be a configured M3DB namespace with the specified resolution/retention or the coordinator will fail to start.
+
+In the situation that you choose to use M3's aggregation functionality, there are a variety of aggregation types you can choose from. For example:
+
+```yaml
+carbon:
+  ingester:
+    listenAddress: "0.0.0.0:7204"
+    rules:
+      - pattern: .*
+        aggregation:
+          type: last
+        policies:
+          - resolution: 1m
+            retention: 48h
+```
+
+The config above will aggregate ingested carbon metrics into `1 minute` tiles, but instead of taking the `mean` of every datapoint, it will emit the `last` datapoint that was received within a given tile's window.
```

Comment:
Hey do you want to list all the aggregation functions, maybe under a subheading `Aggregation` or something?  I'm not bullish on where to but it/etc, but just a list somewhere of what's supported would be good.

Overall this looks great, nice work.